src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Next time don't reset margins or paddings in * selector, it's better to reset them for tags you are using